lib/design/custom_view/agora_notification_card.dart

```diff
@@ -21,6 +24,12 @@ class AgoraNotificationCard extends StatelessWidget {
     return AgoraRoundedCard(
       borderColor: AgoraColors.border,
       padding: EdgeInsets.symmetric(vertical: AgoraSpacings.base, horizontal: AgoraSpacings.base),
+      onTap: () {
+        TrackerHelper.trackClick(
+          clickName: AnalyticsEventNames.notificationClicked,
+          widgetName: AnalyticsScreenNames.notificationPage,
+        );
```

Comment:
J'aurai plutôt fait un `trackEvent` à l'affichage de la notif avec en `widgetName` un truc genre `Notification in-app` et je sais pas pour `eventName` (ou l'inverse)